testspec/src/conf.py

```diff
@@ -57,8 +57,20 @@
 
 # -- Options for botan_src_path extension ------------------------------------
 
-src_ref_reference = '3.0.0'
-# src_ref_check_url = True
+# ATTENTION ON RELEASE
+# ATTENTION ON RELEASE
+# ATTENTION ON RELEASE
+#
+#   Change this to the Botan GitHub tag this document is referring to,
+#   before cutting a revision of the crypto documentation.
+#
+#   TODO: Add a more reliable way or some sort of checklist to avoid
+#         forgetting this before cuttung a release.
+#
+# ATTENTION ON RELEASE
+# ATTENTION ON RELEASE
+# ATTENTION ON RELEASE
+src_ref_reference = 'master'
```

Comment:
I think this can be reverted now.

Reply:
Yep. I'll do that in a final pass over all the documents (potentially even centralized in the repository).